Assets/Scripts/Core/Controllers/GameController.cs

```diff
@@ -16,9 +16,11 @@ public class GameController : MonoBehaviour {
 
 		[SerializeField] private MovementTilemapVisual movementTilemapVisual_;
 		[SerializeField] private MovementTilemapVisual arrowTilemapVisual_;
+		[SerializeField] private MovementTilemapVisual seletorTilemapVisual_;
```

Comment:
seletor? you mean selector?

Reply:
Well yes, but thought I fixed it already.